bin/select_secondary

```diff
@@ -47,7 +50,7 @@ scxdir = _get_scxdir(ns.scnddir)
 if surv != 'main':
     scxdir = os.path.join(scxdir, surv)
 
-scx = select_secondary(ns.priminfodir, sep=ns.separation, scxdir=scxdir, darkbright=True)
+scx = select_secondary(ns.priminfodir, sep=ns.separation, scxdir=scxdir, darkbright=ns.darkbright)
```

Comment:
With my requested default, the `darkbright` kwarg, here would be `darkbright=not(ns.writeall)` or some-such.

Reply:
OK.